src/system/mod.rs

```diff
@@ -78,7 +78,14 @@ impl FileCloser {
 
     /// Close every file descriptor that is not one of the IO streams or one of the file
     /// descriptors passed via [`FileCloser::except`].
-    pub(crate) fn close_the_universe(self) -> io::Result<()> {
+    ///
+    /// # Safety
+    ///
+    /// This method violates [I/O Safety](https://doc.rust-lang.org/std/io/index.html#io-safety)
```

Comment:
```suggestion
    /// Incorrect use of this method can violate [I/O Safety](https://doc.rust-lang.org/std/io/index.html#io-safety)
```
The way it's currently phrased is too strong.